etna/transforms/timestamp/holiday.py

```diff
@@ -10,23 +11,44 @@
 from etna.transforms.base import IrreversibleTransform
 
 
+class HolidayTransformMode(str, Enum):
+    """Enum for different imputation strategy."""
+
+    binary = "binary"
+    category = "category"
+
+    @classmethod
+    def _missing_(cls, value):
+        raise NotImplementedError(
+            f"{value} is not a valid {cls.__name__}. Supported mode: {', '.join([repr(m.value) for m in cls])}"
+        )
+
+
 class HolidayTransform(IrreversibleTransform, FutureMixin):
     """HolidayTransform generates series that indicates holidays in given dataframe."""
 
-    def __init__(self, iso_code: str = "RUS", out_column: Optional[str] = None):
+    NO_HOLIDAY: str = "NO_HOLIDAY"
+
+    def __init__(
+        self, iso_code: str = "RUS", mode: str = HolidayTransformMode.binary, out_column: Optional[str] = None
```

Comment:
May be better to use default value just "binary", because with enum could be problems during `to_dict`. We will address this issue with enums in the future.